src/CONST.ts

```diff
@@ -2326,6 +2326,21 @@ const CONST = {
         },
         CARD_TITLE_INPUT_LIMIT: 255,
     },
+    COMPANY_CARDS: {
+        STEP_NAMES: ['1', '2', '3', '4'],
```

Comment:
We aren't using it and seems not needed bcoz we are not showing the steps header.